cdap-runtime-ext-dataproc/src/main/java/io/cdap/cdap/runtime/spi/provisioner/dataproc/DataprocProvisioner.java

```diff
@@ -323,7 +332,7 @@ String getImageVersion(ProvisionerContext context, DataprocConf conf) {
         default:
           if (context.getAppCDAPVersionInfo() == null ||
             context.getAppCDAPVersionInfo().compareTo(SPARK3_CDAP_DEFAULT) < 0) {
-            imageVersion = "1.3";
+            imageVersion = "1.5";
```

Comment:
I think we can drop it altogether. `SPARK2_2_11` is not really going to be compatible with 1.5 going forward. Just return "2.0". IMHO it's better to do 1.3->2.0 than switch 1.3-(platform upgrade)->1.5-(pipeline upgrade)->2.0